### ISSUE TITLE
[Maybe] Reduce maintenance in BasketballBundService

### ISSUE THREAD

**Issue by renao** [HertenerLoewen/Api#1]
The info scraper looks good to me, thanks :+1:

=> https://github.com/HertenerLoewen/Api/blob/master/app/Services/BasketballBundService.php

Just one quick idea to (possibly) reduce the maintenance costs on this file:

What about using a regex to determine the `$team` name?

I am not that familiar with php but I just got hit by fiddling around with `preg_grep` and got hits with a simple `/(Hertener Löwen)(.*)/` - examplewise:  http://www.phpliveregex.com/p/n4F

Just a tiny refactoring idea ;)

Cheers

**Comment by renao** [comment]
Hmpf, okay, habe gerade erst gesehen, dass du eh auf deutsch die Repos dokumentiert hast.

Wir können dahingehend natürlich auch gerne auf deutsch weitermachen ;-)

**Comment by PascaleBeier** [comment]
@renao 

Wir haben schon das Problem, wenn zwei Mannschaften in der selben Liga spielen, wie H5 und H6. 

Wie wirkt sich das darauf aus? Derzeit habe ich einfach die Fallabfrage gemacht 😄

**Comment by PascaleBeier** [comment]
Wir müssen auch irgendwo die Pfade / Queries / xPaths je Liga angeben.

(Wo finde ich die Spielliste etc)

 Das wollte ich ursprünglich über die Config machen, aber ich halte das für unnötig kompliziert, weil in den nächsten Jahren eh nur diese Liga relevant ist

**Comment by renao** [comment]
Ja, kann ich nachvollziehen. Ich habe bei so hart-verdrahteten Stellen immer leichte Bauchschmerzen, dass es einem später mal auf die Füße fällt.

Ich bin damals davon ausgegangen, dass du ALLE Löwen-Teams haben möchtest, nicht nur für eine Mannschaft - manchmal hilft es aber auch den Code zwei mal zu lesen ;-)

Spontan fällt mir aber auch keine andere Lösung ein, ich lass die Idee ansonsten einfach mal einen Moment bei mir reifen und melde mich sonst wieder.
